src/Servers/HttpSys/samples/TlsFeaturesObserve/Program.cs

```diff
@@ -41,7 +43,14 @@
     success = httpSysPropFeature.TryGetTlsClientHello(bytes, out _);
     Debug.Assert(success);
 
-    await context.Response.WriteAsync($"[Response] connectionId={connectionFeature.ConnectionId}; tlsClientHello.length={bytesReturned}; tlsclienthello start={string.Join(' ', bytes.AsSpan(0, 30).ToArray())}");
+    await context.Response.WriteAsync(
+        $"""
+            connectionId            = {connectionFeature.ConnectionId};
+            negotiated cipher suite = {tlsHandshakeFeature.NegotiatedCipherSuite}; 
```

Comment:
Do you not have tests for these hosting models? Should those tests be updated to expect a non-null answer now?

Reply:
its a new type appeared in net10, and its a last-minute PR to try populate this value.
I dont imagine a nice way to test this, because its interop honestly. I dont see any valuable tests for SNI for instance.